spec/amp-cache-debugging.md

```diff
@@ -0,0 +1,23 @@
+# AMP Cache Debugging
+
+**Why is my doc broken on an AMP cache?**
+
+Valid AMP documents typically appear and behave the same on AMP Caches as they
+do on the origin. However, there are some components and server configurations
+that can be problematic.
+
+If a particular document appears and behaves as expected on your origin, but not
+when viewed via the cache (how to map origin URLs to [Google's AMP Cache](https://developers.google.com/amp/cache/overview#amp-cache-url-format)),
+first open the developer console in your browser, and ensure no errors or
+warnings appear. If this doesn't resolve the problem, run the document through
+[AMPBench](https://ampbench.appspot.com/) and resolve any unexpected errors or
+warnings. If you still have a problem after following these steps, check the
+table below.
+
+|Symptom|Issue|Solution|
+|---|---|---|
+|Web fonts do not appear (fallback fonts are used)|Fonts served with incorrect MIME type|Specify an [acceptable MIME type](https://github.com/ampproject/amphtml/blob/master/spec/amp-cache-guidelines.md#mime-types-for-fonts) for fonts|
```

Comment:
The same can also happen with images. Either a non `image/` mime type or the wrong mime type.